### DISCUSSION TITLE
Usage of more then one source to document files

### DISCUSSION THREAD

**Discussion by Timbuktu1982** [EnterTheNameHere/esdoc-monorepo#7] [Q&A] [answered]
Hey there,
is it possible at the moment to configure esdoc for more then one file source (eg. "source": "./app, ./blueprints" OR "source": ["./app", "./blueprints"]?

**Comment by EnterTheNameHere** [comment]
No, currently it treats source as a directory, so it's not possible. I'll see if I can make it array.

**Reply by Timbuktu1982** [reply]
Cool, thank you. If you have not a lot of time, i can support you.

**Comment by EnterTheNameHere** [comment, marked as answer]
Implemented in v2.4.0

You can now use
```
{
    source: [ "directory", "anotherDirectory", "another/directory/deep/" ]
}
```

However tested only under same root directory, manually. I'll need to write unit tests for it, but that will need some refactoring first.

**Reply by Timbuktu1982** [reply]
Hey EnterTheNameHere,

thank you so much. 

It works perfectly...
